### DISCUSSION TITLE
gevent_wsgi is giving ImportError: cannot import name 'selectors'

### DISCUSSION THREAD

**Discussion by arun6894** [miguelgrinberg/python-socketio#1020] [Q&A] [answered]
I have created the server with **async_mode='gevent_uwsgi'**  with gevent.monkey.patch_all() as well.
I get to see the following error when debugging at engineio/server.py for the line below.
```

self._async = importlib.import_module(
                    'engineio.async_drivers.' + mode)._async
``` 
On checking the **engineio/async_drivers/gevent_uwsgi.py**. I could see the selectors module is not being recognized in VS code where other imports like Event and Queue were.

I could see the selectors module existed in gevent but why isn't it recognized in the gevent_uwsgi file?

Using - python3.6.8
python-engineio==4.3.2
python-socketio==5.6.0
gevent==1.2.2
uWSGI==2.0.20
gevent-websocket==0.10.1

**Comment by miguelgrinberg** [comment, marked as answer]
Try upgrading gevent to a recent version.
